diff --git a/linux-odroid-c2/PKGBUILD b/linux-odroid-c2/PKGBUILD
index a27b8d3..c1ade44 100644
--- a/linux-odroid-c2/PKGBUILD
+++ b/linux-odroid-c2/PKGBUILD
@@ -156,7 +156,7 @@ _package-headers() {
 
   mkdir -p "${pkgdir}/usr/lib/modules/${_kernver}/build/include"
 
-  for i in acpi asm-generic config crypto drm generated keys linux math-emu \
+  for i in acpi asm-generic clocksource config crypto drm generated keys linux math-emu \
     media net pcmcia scsi sound trace uapi video xen; do
     cp -a include/${i} "${pkgdir}/usr/lib/modules/${_kernver}/build/include/"
   done
