diff -Naur a/lib/libspl/include/sys/types.h b/lib/libspl/include/sys/types.h
--- a/lib/libspl/include/sys/types.h	2016-03-18 07:56:36.793553352 +0000
+++ b/lib/libspl/include/sys/types.h	2016-03-18 07:57:19.354199034 +0000
@@ -30,7 +30,6 @@
 #include <sys/isa_defs.h>
 #include <sys/feature_tests.h>
 #include_next <sys/types.h>
-#include <sys/param.h> /* for NBBY */
 #include <sys/types32.h>
 #include <sys/va_list.h>
 
@@ -95,4 +94,6 @@
 } lloff_t;
 #endif
 
+#include <sys/param.h> /* for NBBY */
+
 #endif
